cds/modules/records/serializers/schemas/project.py

```diff
@@ -64,8 +63,7 @@ class ProjectSchema(StrictKeysSchema):
     _deposit = fields.Nested(ProjectDepositSchema, required=True)
     _cds = fields.Nested(_CDSSSchema, required=True)
     title = fields.Nested(TitleSchema, required=True)
-    description = fields.Nested(DescriptionSchema, required=True)
-    date = DateString(required=True)
+    description = fields.Str()
```

Comment:
from project has been removed the [required](https://github.com/CERNDocumentServer/cds-dojson/blob/qa/cds_dojson/schemas/records/videos/project/project-v1.0.0.json).